lms/djangoapps/courseware/tests/test_submitting_problems.py

```diff
@@ -459,33 +459,6 @@ def test_show_answer_doesnt_write_to_csm(self):
         csmh = BaseStudentModuleHistory.get_history(student_module)
         self.assertEqual(len(csmh), 3)
 
-    def test_grade_with_max_score_cache(self):
-        """
-        Tests that the max score cache is populated after a grading run
-        and that the results of grading runs before and after the cache
-        warms are the same.
-        """
```

Comment:
We have pretty good coverage on the Block Structure layer itself.  However, we rely on existing grading tests to find issues with converting over to using Block Structures - since the high layer interface remained the same - and only the lower level implementation changed.  So I am inclined to keep these tests as they seem to be testing the grading interface as well.  Unless there's already other existing tests that test the same thing - and not specific to `MaxScoresCache`.